src/services/etherspot.js

```diff
@@ -596,7 +596,7 @@ export class EtherspotService {
     }
 
     // submit current batch
-    const { hash: batchHash } = await sdk.submitGatewayBatch();
+    const { hash: batchHash } = await sdk.submitGatewayBatch({ guarded: false });
```

Comment:
Just for my information. what is the use of **guarded**

Reply:
It prevents users absuing our services and ensures that transactions are being sent through our relayers.